Source/Core/DiscIO/DirectoryBlob.cpp

```diff
@@ -930,6 +951,15 @@ DirectoryBlobPartition::DirectoryBlobPartition(
   BuildFST(std::move(nodes), new_fst_address);
 }
 
+DirectoryBlobPartition::DirectoryBlobPartition(const DirectoryBlobPartition& rhs)
+    : m_contents(rhs.m_contents), m_disc_header(rhs.m_disc_header), m_bi2(rhs.m_bi2),
+      m_apploader(rhs.m_apploader), m_fst_data(rhs.m_fst_data), m_key(rhs.m_key),
+      m_root_directory(rhs.m_root_directory), m_is_wii(rhs.m_is_wii),
+      m_address_shift(rhs.m_address_shift), m_data_size(rhs.m_data_size),
+      m_wrapped_partition(rhs.m_wrapped_partition)
```

Comment:
Similar to above, `m_disc_header`, `m_bi2`, and `m_apploader` should be able to be turned into local variables. `m_fst_data` can also become a local variable if you just pass a pointer to it to `DirectoryBlobPartition::WriteEntryData` and `DirectoryBlobPartition::WriteEntryName`.

Reply:
m_disc_header stays. It's used in quite a few places outside of this and things keep reading from it and writing to it. It's becoming far too much of a hassle to remove it.

Reply:
Agreed, let's skip `m_disc_header`.